src/Components/IndexScene/IndexScene.tsx

```diff
@@ -62,7 +61,8 @@ export class IndexScene extends SceneObjectBase<IndexSceneState> {
     super({
       $timeRange: state.$timeRange ?? new SceneTimeRange({}),
       $variables:
-        state.$variables ?? getVariableSet(state.initialDS ?? getLastUsedDataSourceFromStorage(), state.initialFilters),
+        state.$variables ??
+        getVariableSet('grafanacloud-logs' ?? getLastUsedDataSourceFromStorage(), state.initialFilters),
```

Comment:
What would happen if, for whatever reason, there's no `grafanacloud-logs` in the instance? I think the first use could be one example, where you don't have either `grafanacloud-logs` and last used from storage.

Reply:
It just selects the first from the list.

![image](https://github.com/grafana/explore-logs/assets/8092184/940ccd0d-cb0a-405b-a345-8365d538e69a)
-> 
![image](https://github.com/grafana/explore-logs/assets/8092184/efbf894d-2e9d-42ac-8507-5262bf982df4)

Reply:
Fantastic!